### PR TITLE
Update links in documentation

### PR DESCRIPTION
**Pull request by sudipt1999**
Almost all links are updated ! Still a second cross check would be helpful!
https://github.com/taskcluster/taskcluster-lib-azure/releases in azure README.md
https://github.com/taskcluster/schema-validator-publisher/blob/master/package.json in valida README.md
these links are still need to be updated ! couldn't find the updated links!

### CODE REVIEW

File: clients/client/README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ let pc = await pulse.consume({
 
 The set of API entries listed below is generated from the built-in references.
 Detailed documentation with description, payload and result format details is
-available in the [docs reference section](https://docs.taskcluster.net/docs/reference).
+available in the [docs reference section](https://github.com/taskcluster/taskcluster/tree/master/libraries/references).
```
Comment (djmitche) [inline clients/client/README.md]: The taskcluster-lib-references library and the "reference section" are different.  I think we can just drop the link here, leaving something like "..available in the reference section of the Taskcluster documentation"

File: infrastructure/terraform/README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ in the shell you are applying this from.
 ## Requirements not managed here
 
 - A kubernetes cluster
-- An (nginx ingress controller)[https://kubernetes.github.io/ingress-nginx/deploy/] in said cluster
+- An [nginx ingress controller](https://kubernetes.github.io/ingress-nginx/deploy/) in said cluster
```
Comment (djmitche) [inline infrastructure/terraform/README.md]: good fix :)

File: libraries/references/README.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ during the cluster build process:
 ```
 
 This is a subset of the
-[taskcluster-lib-docs](https://github.com/taskcluster/taskcluster-lib-docs)
+[taskcluster-lib-docs](https://github.com/taskcluster/taskcluster/tree/master/libraries/loader)
```
Comment (djmitche) [inline libraries/references/README.md]: docs, not loader

File: services/auth/README.md [hunk 4]
```diff
@@ -36,4 +36,4 @@ To do so, copy `user-config-example.yml` to `user-config.yml` and fill in the ne
 Taskcluster team members can provide you with some testing-only credentials -- just ask, and provide a GPG key (use https://keybase.io if you don't have one).
 You can get your own pulse credentials at https://pulseguardian.mozilla.org.
 
-The taskcluster team has a series of [best practices](/docs/manual/devel/best-practices) which may help guide you in modifying the source code and making a pull request.
+The taskcluster team has a series of [best practices](https://github.com/taskcluster/taskcluster/tree/master/dev-docs) which may help guide you in modifying the source code and making a pull request.
```
Comment (djmitche) [inline services/auth/README.md]: please add `/best-practices` to the end here

File: services/web-server/README.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ by the web application.
 
 ## Configuration
 
-Configuration is done via [taskcluster-lib-config](/librarices/config) like all
+Configuration is done via [taskcluster-lib-config](https://github.com/taskcluster/taskcluster/tree/master/libraries/loader) like all
```
Comment (djmitche) [inline services/web-server/README.md]: `/config`, not `/loader` :)

**Comment by sudipt1999** [comment]
Sure sir ! Wont be a problem .

**Pull request merged**